### ISSUE TITLE
New error: Cannot read property 'fire' of undefined

### ISSUE THREAD

**Issue by findel** [ShyykoSerhiy/skyweb#62]
This error occurs every few hours since the last update:

```
/home/projects/bot/node_modules/skyweb/dist/src/polling/poll.js:27
                    _this.eventEmitter.fire('error', 'Failed to poll messages.' +
                                      ^

TypeError: Cannot read property 'fire' of undefined
    at Request._callback (/home/projects/bot/node_modules/skyweb/dist/src/polling/poll.js:27:39)
    at Request.self.callback (/home/projects/bot/node_modules/skyweb/node_modules/request/request.js:187:22)
    at emitTwo (events.js:87:13)
    at Request.emit (events.js:172:7)
    at Request.<anonymous> (/home/projects/bot/node_modules/skyweb/node_modules/request/request.js:1044:10)
    at emitOne (events.js:77:13)
    at Request.emit (events.js:169:7)
    at IncomingMessage.<anonymous> (/home/projects/bot/node_modules/skyweb/node_modules/request/request.js:965:12)
    at emitNone (events.js:72:20)
    at IncomingMessage.emit (events.js:166:7)
```

**Comment by ShyykoSerhiy** [comment]
oh, 💩

**Comment by ShyykoSerhiy** [comment]
fixed in 0.3.1
